src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx

```diff
@@ -74,31 +72,20 @@ export class SceneByVariableRepeaterGrid extends SceneObjectBase<SceneByVariable
     return layout === LayoutType.ROWS ? GRID_TEMPLATE_ROWS : GRID_TEMPLATE_COLUMNS;
   }
 
-  protected _variableDependency: VariableDependencyConfig<SceneByVariableRepeaterGridState> =
```

Comment:
Not needed anymore, we rely on the Scenes mechanism to update things automatically whenever a variable that is present in a query changes.